TestON/drivers/common/cli/quaggaclidriver.py

```diff
@@ -23,7 +23,20 @@ def __init__( self ):
     def connect( self, **connectargs ):
         for key in connectargs:
             vars( self )[ key ] = connectargs[ key ]
+        self.name = self.options[ 'name' ]
```

Comment:
It looks ok, when I move to onos1.1 branch I will pay attention to those changes. Thanks.